src/Chart.ts

```diff
@@ -99,10 +129,15 @@ export function impactChartConfig(filterData: FilteredData,
         }
     };
 }
-
+// this should return a ChartConfiguration object but that doesn't have the
+// annotation entry. So I've set it to any until I work out how to modify
+// ChartConfiguration in a way that doesn't break everything!
```

Comment:
The simplest solution is probably just to define:
```
export interface ChartOptionsWithAnnotation extends ChartOptions {
    annotation: any;
}

export interface AnnotatedChartConfiguration extends ChartConfiguration {
    options: ChartOptionsWithAnnotation;
}
```
Then the return type of `timeSeriesChartConfig` can be `AnnotatedChartConfiguration` (and like you say, when you have time you can add a type def for the `annotation` property too.)